<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDBWindow</class>
 <widget class="QWidget" name="SettingDBWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки подключения к БД</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/rec/img/ico.ico</normaloff>:/rec/img/ico.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="port_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_name_user">
       <property name="text">
        <string>Имя пользователя</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="user_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_name_db">
       <property name="text">
        <string>Название базы данных</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="name_db_edit"/>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="btn_connect">
       <property name="styleSheet">
        <string notr="true">QPushButton:hover { background-color: rgb(186, 189, 182) }
QPushButton:pressed { background-color: rgb(85, 87, 83) }</string>
       </property>
       <property name="text">
        <string>Подключиться</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_port">
       <property name="text">
        <string>Порт</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_name_host">
       <property name="text">
        <string>Имя хоста</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="host_name_edit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="password_edit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
